pkg/registry/core/endpoint/registry.go

```diff
@@ -66,7 +66,7 @@ func (s *storage) GetEndpoints(ctx genericapirequest.Context, name string, optio
 }
 
 func (s *storage) UpdateEndpoints(ctx genericapirequest.Context, endpoints *api.Endpoints) error {
-	_, _, err := s.Update(ctx, endpoints.Name, rest.DefaultUpdatedObjectInfo(endpoints))
+	_, _, err := s.Update(ctx, endpoints.Name, rest.DefaultUpdatedObjectInfo(endpoints), rest.ValidateAllObjectFunc, rest.ValidateAllObjectUpdateFunc)
```

Comment:
only safe because of how it is called.  Needs a comment.

Reply:
Fixed